packages/blade/src/components/Input/OTPInput/_decisions/_decisions.md

```diff
@@ -0,0 +1,92 @@
+# OTPInput Decisions <!-- omit in toc -->
+
+This doc talks about the API decisions for `OTPInput`. 
+## Index <!-- omit in toc -->
+
+- [OTPInput API](#otpinput-api)
+  - [Sample Usage](#sample-usage)
+- [Keyboard return key types for web and native](#keyboard-return-key-types-for-web-and-native)
+- [OTP SMS auto-read feature](#otp-sms-auto-read-feature)
+  - [Web](#web)
+  - [Native Android](#native-android)
+  - [Native iOS](#native-ios)
+
+## OTPInput API
+
+<img src="./otp.png" width="300" alt="otp input" />
+
+### Sample Usage
+
+```tsx
+<OTPInput
+  otpLength={4}
+  label="Enter One Time Code"
+  labelPosition="left"
+  helpText="Enter the OTP sent to your phone number"
+  errorText="Invalid OTP"
+  successText="Valid OTP"
+  validationState={inputValue.length < 4 ? 'error' : 'success'}
+/>
+```
+
+
+| Prop | Type | Required | Default | Description |
+|---|---|---|---|---|
+| otpLength | `4`, `6`  | No | `4` | Determines the length of the OTP input. |
```

Comment:
How can we be certain this would always be 4 or 6. This depends on backend service right?

Reply:
In our use-cases its 4 or 6 and that's what the design is also aligned to. If we see more use-cases, we can always add more or open this up to be a number.